cv32e40x/tests/programs/custom/cv32e40x_csr_access_test/test.yaml

```diff
@@ -2,3 +2,4 @@ name: cv32e40x_csr_access_test
 uvm_test: uvmt_$(CV_CORE_LC)_firmware_test_c
 description: >
     CSR access test for the cv32e40x
+llvm_cflags: -fno-integrated-as
```

Comment:
What happens if this test-program is compiled with gcc?  Will the scriptware be smart enough to ignore these flags?